front/learn-next-js/src/app/components/ScheduleTable.tsx

```diff
@@ -0,0 +1,45 @@
+import { promises as fs } from 'fs';
+
+
+async function CreateTable(schedule: {[date: string]: {[shift: string]: string}}) {
```

Comment:
you can define the type for `schedule` as an interface. read up on it and re-use it